### PR TITLE
Loading "sun" works in a right way in catalog page

### PR DESCRIPTION
**Pull request by ValDekh**
Change loading "sun" behavior in catalog page. Not it works in a right way. Issue: #909

### CODE REVIEW

**Review by Maxym1507 — APPROVED** [review]

**Comment by KateYatsiuk** [comment]
Please, change merge branch to **develop**

**Comment by ValDekh** [comment]
Changed branch to develop

File: src/app/stores/streetcodes-catalog-store.ts [hunk 1]
```diff
@@ -6,22 +6,25 @@ import { StreetcodeCatalogRecord } from '@/models/streetcode/streetcode-types.mo
...
     public catalog = new Array<StreetcodeCatalogRecord>();
 
-    public isNotEightorZero = false;
+    public moreThenEight = false;
 
     constructor() {
         makeAutoObservable(this);
     }
 
     public fetchCatalogStreetcodes = async (page: number, count = 8) => {
```
Comment (KateYatsiuk) [inline src/app/stores/streetcodes-catalog-store.ts:15]: Let's move 8 to const to avoid magic numbers

**Review by KateYatsiuk — APPROVED** [review]
Works fine!)

**Review by AndriiS1 — APPROVED** [review]

File: src/app/stores/streetcodes-catalog-store.ts (hunk 1 above)
Comment (ValDekh) [reply src/app/stores/streetcodes-catalog-store.ts:15]: I believe that 8 in its place and shows how much will be fetched. Seeing it separately in constant will worsen the readability of the code.

Comment (KateYatsiuk) [reply src/app/stores/streetcodes-catalog-store.ts:15]: Well, it's your choice, I will just express y opinion. You have at least 2 usings of digit 8 in your code, not only one. So in what way will the readability worsen if you name the constant properly? Imagine that later smth else will need to be added to this component using the limitation 8. In this way, a constant will immediately catch the eye of the dev. The code should be easy for other developers to maintain.
But of course it's not critical, up to you)
Good job!

Comment (ValDekh) [reply src/app/stores/streetcodes-catalog-store.ts:15]: Maybe you are right. I will change it as you proposed ;)

Comment (ValDekh) [reply src/app/stores/streetcodes-catalog-store.ts]: Change. At last you will be happy

**Review by mykola-krai — APPROVED** [review]

**Pull request merged**